protobuf-definitions/proto/mlagents_envs/communicator_objects/capailities.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+
+option csharp_namespace="MLAgents.CommunicatorObjects";
+package communicator_objects;
+
+/*
+ * A Capabilities message that will communicate both C# and Python
+ * what features are available to both.
+ */
+message UnityRLCapabilities {
+    // These are the 1.0 capabilities.
+    bool BaseRLCapabilities = 1;
```

Comment:
nit: camel_case for field names (the code generator will convert to PascalCase for C# later)

Reply:
ok